polars/lazy/expr/index.ts

```diff
@@ -853,7 +853,7 @@ export const _Expr = (_expr: any): Expr => {
       return _Expr(_expr.last());
     },
     list() {
-      return _Expr(_expr.list());
+      return _Expr(_expr);
```

Comment:
This one I was not sure about the [test](https://github.com/Bidek56/nodejs-polars/blob/1cb126640eecc421b90b9f9dd7017b140edb8d50/__tests__/expr.test.ts#L475) because this code:
`const expected = pl.DataFrame({list: ["a", "b", "c"],});`
produces: 

```
shape: (3, 1)
┌──────┐
│ list │
│ ---  │
│ str  │
╞══════╡
│ a    │
├╌╌╌╌╌╌┤
│ b    │
├╌╌╌╌╌╌┤
│ c    │
└──────┘
```
not a 
```
┌─────────────────┐
│ list            │
│ ---             │
│ list[str]       │
╞═════════════════╡
│ ["a", "b", "c"] │
└─────────────────┘
```

Reply:
hmm yeah that doesn't seem right. 

the expected should be 
```js
    const expected = pl.DataFrame({
      list: [["a", "b", "c"]],
    });
```
`list` was recently renamed to `implode` in upstream polars. Since this PR is updating the polars dependency, maybe we should also include those changes?

Reply:
This PR updates rs-polars to 0.28. ATM my `cargo` is not able to build with 0.30.0.
Once I figure out my `cargo` issue, I will do another PR for rs-polars 0.30+